server/server.go

```diff
@@ -102,7 +102,10 @@ type UserConfig struct {
 	RepoWhitelist          string `mapstructure:"repo-whitelist"`
 	// RequireApproval is whether to require pull request approval before
 	// allowing terraform apply's to be run.
-	RequireApproval        bool            `mapstructure:"require-approval"`
+	RequireApproval bool `mapstructure:"require-approval"`
+	// RequireMergeable is similar to RequireApproval, but won't allow running
+	// `terraform apply` unless the PR is actually mergable.
```

Comment:
Change to:
```
// RequireMergeable is whether to require pull requests to be mergeable before
// allowing terraform apply's to run.
```

I prefer not having to refer to another struct field's docs if I can help it, need to fix mispelling of mergeable, and I try to avoid the backticks since they're not parsed by godoc